src/sentry/api/endpoints/project_keys.py

```diff
@@ -27,10 +29,16 @@ def create_key_scenario(runner):
     )
 
 
+class RateLimitSerializer(serializers.Serializer):
+    count = EmptyIntegerField(min_value=0, required=False, allow_null=True)
+    window = EmptyIntegerField(min_value=0, max_value=60 * 60 * 24, required=False, allow_null=True)
```

Comment:
Could we import/reuse this serializer from the project_key_details endpoint? Perhaps this serializer and the ones defined in project_key_details could be moved to a module together?

Reply:
This is indeed a bad case of copy-paste.
I'll take a look at the codebase and try to find some instances of shared serializers to try and stay consistent